CHANGELOG.rst

```diff
@@ -298,7 +298,7 @@ All notable changes to this project are documented in this file.
 
 
 
-[0.4.9] 2018-02-21
+[0.4.7] 2018-02-21
```

Comment:
hm... please don't change all the release numbers! the changelog entries are all related to the specific release numbers, these changes would mess this up completely :P